app/app.go

```diff
@@ -90,7 +90,7 @@ type NibiruApp struct {
 	AppKeepers // embed all module keepers
 
 	// the module manager
-	mm *module.Manager
+	ModuleManager *module.Manager
```

Comment:
do we have some benefit of adding it public?

Reply:
It's required for `TestAppImportExport`.